spec/order_spec.rb

```diff
@@ -0,0 +1,66 @@
+require 'order'
+
+describe Order do
+  let(:restaurant) {double (:restaurant), :menu => menu, :check_inclusion? => true, :get_price => 10}
```

Comment:
Line is too long. [101/80]<br>Space missing inside {.<br>Space missing inside }.